app/index.js

```diff
@@ -22,10 +23,47 @@ const finalState = Immutable(initialStoreState).merge([initialServerState, initi
   deep: true,
 });
 const store = configureStore(finalState);
+const warningStyle = {
+  margin: '20px',
+  padding: '20px',
+  color: '#5d3d08',
+  backgroundColor: '#fcf8e3',
+  border: '2px solid #faebcc',
+};
 
-render(
-  <Provider store={store}>
-    <Router>{getRoutes()}</Router>
-  </Provider>,
-  document.getElementById('root')
-);
+if (browserName === 'IE') {
+  render(
+    <div>
+      <p style={warningStyle}>
+        Currently, Varaamo does not support IE11.
```

Comment:
That is no longer true after adding the polyfill.

Reply:
The error message `div` is getting rendered in IE browser and normally as before in other browsers only after adding the polyfill. Without polyfill, I am just getting the blank screen in IE.